src/UserManagerSettings.ts

```diff
@@ -10,6 +10,7 @@ export const DefaultPopupWindowFeatures: PopupWindowFeatures = {
     location: false,
     toolbar: false,
     height: 640,
+    closePopupWindowAfter: 0,
```

Comment:
- please add the unit of time to the name `closePopupWindowAfter` -> `closePopupWindowAfterInSeconds` 
- maybe -1 is more readable?